src/main/kotlin/edu/carroll/polyominoes/web/rest/controller/GameRestfulController.kt

```diff
@@ -0,0 +1,50 @@
+package edu.carroll.polyominoes.web.rest.controller
+
+import edu.carroll.polyominoes.service.game.GameService
+import edu.carroll.polyominoes.web.rest.ajax.GameStats
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@EnableWebSecurity
+@RestController
+@RequestMapping("/api/game")
+class GameRestfulController(private val gameService: GameService) {
+
+    companion object {
+        private val log = LoggerFactory.getLogger(GameRestfulController::class.java)
+    }
+
+    @PostMapping(consumes = ["application/json"])
+    @ResponseStatus(HttpStatus.OK)
+    fun getStatsPost(@RequestBody stats: GameStats): Boolean {
+
+        val authentication = SecurityContextHolder.getContext().authentication
+
+        if (authentication == null) {
```

Comment:
Is the validation code for authentication necessary when using the @EnableWebSecurity annotation?